syncd/syncd_notifications.cpp

```diff
@@ -1,8 +1,13 @@
 #include "syncd.h"
 #include "sairedis.h"
 
-// mutex to protect notification send call
-std::mutex g_ntf_mutex;
+#include <queue>
+#include <memory>
+#include <condition_variable>
```

Comment:
include those standard headers at very beginning

Reply:
common practice is to put std libs as last includes and not put everything into one header to reduce compilation time of source and remove headers that are not needed cor specific source file